README.md

```diff
@@ -5,22 +5,24 @@ Automating the boring stuff.
 
 Managing a website can be difficult. You can end up with hundreds of pages, and making sure all of them are functional after an event, say a deployment, peaks of traffic, editorial changes, ads or javascripts behaving badly, ... Creating software and maintaining websites and apps is hard... regression testing should not.
 
-# Requirements
+Before you start, Clone / fork this repository and go into the folder.
+There are two ways of using this tool: With [Vagrant](#Vagrant) or [Docker](#Docker).
 
+
+# Vagrant
+Download and install [Vagrant](https://www.vagrantup.com/downloads).
+
+## Requirements
 - PHP 7.4
 - Sqlite
 - A (maybe big) list of urls to crawl. Robots.txt and Sitemaps detected automatically
 
-Alternatively, using the Vagrant machine provided will take care of the sqlite and PHP 7.4 requirements. Keep reading.
-
-# Install
-
-1. Clone / fork this repository.
+## Installation
 
-2. Run:
+1. Run:
```

Comment:
We seem to have ended up with a bunch of "step 1" in this file?

Reply:
Yes, its MD language, it does the bullet numbers automatically, see the preview of Readme here https://github.com/alex-moreno/glitcherbot/blob/d5e68d96070552570bae835b12d26540b990909b/README.md

Reply:
Gotcha, thanks @marcelovani - it wasn't obvious from the diff